spark/src/test/scala/org/apache/spark/sql/CometTPCDSQueryTestSuite.scala

```diff
@@ -213,7 +221,7 @@ class CometTPCDSQueryTestSuite extends QueryTest with TPCDSBase with CometSQLQue
           // that can cause OOM in GitHub Actions
           if (!(sortMergeJoin && name == "q72")) {
             System.gc() // Workaround for GitHub Actions memory limitation, see also SPARK-37368
-            runQuery(queryString, goldenFile, conf)
+            runQuery(queryString, goldenFile, baseConf ++ conf)
```

Comment:
This is where COMET_ENABLED gets passed in

Reply:
There are `tpcdsQueriesV2_7_0` and `tpcdsExtendedQueries` as well. Should we add `baseConf` to `joinConfs` so that all confs get the `baseConf`?

Reply:
> Should we add `baseConf` to `joinConfs` so that all confs get the `baseConf`?

That is what I initially tried, but that would also have required updates to CI scripts (`benchmark.yml`) because there has to be an exact match to the configs defined there, such as:

```
          SPARK_TPCDS_JOIN_CONF: |
            spark.sql.autoBroadcastJoinThreshold=-1
            spark.sql.join.preferSortMergeJoin=true
```

Reply:
I have now added `baseConf` to the calls for `tpcdsQueriesV2_7_0` and `tpcdsExtendedQueries`